functions/greater-than-ten/greater.js

```diff
@@ -0,0 +1,23 @@
+/*function checkValue(highNumber) {
+  if (highNumber > 10) {
+    return true;
+  } else {
+    return "sorry low number";
+  }
+}
+highNumber = 12;
+console.log(highNumber);
+alert(highNumber);
+*/
+
+checkValue = x => {

```

Comment:
missing variable assignment ie `const checkValue = ...`

Reply:
amended the code